script/build.js

```diff
@@ -34,7 +34,7 @@ liquid.filters.humanizeDate = dt => moment(dt).format('MMMM D, YYYY');
 // Set up Metalsmith. BE CAREFUL if you change the order of the plugins. Read the comments and
 // add comments about any implicit dependencies you are introducing!!!
 //
-smith.source(`${BUILD_OPTIONS.contentRoot}/pages`);
+smith.source(`${BUILD_OPTIONS.contentPagesRoot}`).ignore('.git');
```

Comment:
Do we need to conditionally switch between the different content roots depending on if we're on WBC or not? Presumably we're not pulling out both sets of content.

Reply:
There's an override in the `options` file so that WBC will look at the sibling `vagov-content` directory while the Vets.gov builds will continue to look at the `content` child directory